core/trino-main/src/main/java/io/trino/memory/MemoryManagerConfig.java

```diff
@@ -65,6 +66,20 @@ public Duration getKillOnOutOfMemoryDelay()
         return killOnOutOfMemoryDelay;
     }
 
+    @Config("query.low-memory-kill.after-kill-delay")
```

Comment:
low-memory-kill -> low-memory-killer? To be consistent with the config below.

Reply:
gone now